aip/0002.md

```diff
@@ -44,16 +44,30 @@ Currently recognized blocks of AIP numbers are:
 - **1-99:** Reserved for meta-AIPs (generally process-related).
 - **100-999:** General API design guidance
 
-### Google Product Areas
-
-- **2700-2799:** Apps (GSuite)
-- **2500-2599:** Cloud
-- **3000-3099:** Actions on Google
-- **4600-4699:** Geo
-
 To request a block for a specific team that is publishing API guidance or
 documentation germane to that specific team, reach out to api-editors@.
 
+### Custom AIPs
+
+If you creating your own set of AIPs for your company or organization and want
+to maintain compatibility with aip.dev, you will need to prefix your custom AIP
+IDs with a unique prefix. This will ensure that additional AIPs added to
+aip.dev will not conflict with any custom AIPs you might add.
+
+For example. assume you write a new AIP for state management in your AIPs, and
+pick the number `9900`. Instead of calling this `AIP-9900` you should use your
+own custom prefix, like `ACU-9900` (i.e. AIP CUstom). If, in the future,
+aip.dev decides to create a new AIP and use the number `9900`, you could
+continue to implement that AIP in your organization under `AIP-9900` without
+conflicting with your custom `ACU-9900`.
+
+When extending a custom AIP outside of aip.dev, it is important that you link
+to the original AIP. For example, if another company wanted to adopt `ACU-9900`
+in their AIPs, they **must** keep the same ID and prefix, and they **must**
+state the AIP they are extending in the introduction. They **must** also choose
+a different three letter prefix for any custom AIPs that are unique to their
+custom AIPs.
+
```

Comment:
Tried to write out what an org level custom AIP with it's own prefix would look like and how it would work.